decent-dao/fractal-interface
# Ukrainian translation

## Description

This PR translates application to Ukrainian following our script - https://github.com/decent-dao/fractal-interface/blob/develop/TRANSLATIONS.md

## Notes

n/a

## Issue / Notion doc (if applicable)

n/a

## Testing

1. Open the app
2. Switch the language to Ukrainian
3. Verify that you see Ukrainian language
4. Thoroughly walk through the app - check that all the strings are translated (some names like "Fractal", "DAO", "NFT", "Gnosis", "Safe" are not translatable).
5. Glory to Ukraine! Slava Ukraine!

## Screenshots (if applicable)
<img width="1680" alt="Знімок екрана 2022-12-14 о 20 06 57" src="https://user-images.githubusercontent.com/25801162/207674843-4ad0ab1f-0f07-4205-9c63-fe3a2d5e3927.png">
<img width="1680" alt="Знімок екрана 2022-12-14 о 20 10 01" src="https://user-images.githubusercontent.com/25801162/207674850-e7ce1eb1-038e-48bb-9504-f92ee5d07ef1.png">
